### ISSUE TITLE
Language Support Mimicking What Darkest Dungeon Provides

### ISSUE THREAD

**Issue by Hypocrita20XX** [Hypocrita20XX/Conexus#6]
At this point, I don't know how I would even implement such a feature, or if it's necessary, however I think that it might be a good idea eventually. There's also the issue of the quality of translation, as I only speak English and would initially have to use Google Translate, which is not ideal.
Regardless, it's something that I feel needs to be considered and made public.

**Comment by Hypocrita20XX** [comment]
Been a while since this issue was posted, so here's my current (2-2021) thought concerning this.

Most likely I'll provide some kind of functionality as the very last feature I implement for Conexus, which will read from a file and translate the UI according to that file. Ideally this will also translate the log.
This wouldn't necessarily be an end-user, ready-to-use feature, more for advanced users who are bilingual and want to support such a feature for other users of that language. I would provide samples, in English, and then the translations would be read into Conexus in a specific order of some kind.

I won't personally provide these translation files, other than English, which will be the default and created at the start, and this translation file would be placed alongside that, perhaps out of necessity named translation.txt or something similar.

This is, again, most likely the very last thing I'll implement before I sign off on Conexus, so please don't expect this any time soon, if at all.
I'm most concerned with overall stability and simplicity of functionality at this point. With this one possible exception, Conexus is feature-complete. 
Even if I do make this feature, I probably won't refine it too much, and leave it to others to do that as they see fit.

My goal with Conexus has always been to create an easy-to-easy, simple program to organize one's mods, and as of 1.3.0, I've pretty much gotten there. Language support is, all in all, superfluous to that.
But I nevertheless understand that English is not the end-all, be-all standard in all parts of the world, and if it's at all possible, I'll make this happen in some way.

Lastly, this wouldn't necessarily be limited to just what Darkest Dungeon supports. The feature would be agnostic and load whatever translation you provide.

**Comment by Hypocrita20XX** [comment]
Maybe final update concerning the planning side of this.

With 1.4.0, I'm going to consider the main branch feature-complete, so nothing aside from bug fixes from here on for that version.

I would really like to look into this though, so if I somehow pull it off, that will be in its own experimental branch. Not sure I'll put releases of this version in the release tab, I would rather not confuse anyone. I also have no idea how that all works, so... We'll see. 

TLDR: 1.4.0 is feature complete and the main version is what it is, sans translation features. Experimental version might happen later though (v1.5.0 I guess? 1.4.0B? Semantics, a problem for another time.)

**Comment by Hypocrita20XX** [comment]
As much as I would like to do this, I honestly just need to be done with this project.
Been at this for months now, and mentally I'm spent.

So, translations, if they happen, I won't be the guy to make it happen.
Sorry about that.
